pkg/oauthflow/flow.go

```diff
@@ -36,6 +37,11 @@ const htmlPage = `<html>
 </html>
 `
 
```

Comment:
Perhaps we should add some comments to these (as well as updating the names) to denote that they are only meant to be used with `oauth2.sigstore.dev` ; otherwise this LGTM

Reply:
I updated a bit of names and comments
Did you mean renaming the Default[provider]IDTokenGetter as well?
Is there any specific naming prefix you would rather?
PublicInstance, SigstoreDev, .. ?

Reply:
I'm fine with `PublicInstance` as you propose here. I think with renaming the IDTokenGetters we'd be ready to merge this. Thanks for the persistence on getting this to the finish line :)

Reply:
No problem, its my pleasure thanks for the review.
Its my first contribution slowly i hope to have help out.
If you like you can take a look at other PR's i have open on cosign and go-securesystemslib.